podcast/the-changelog-330.md

```diff
@@ -128,7 +128,7 @@ For data analysts - if you tell a data analyst "Oh yeah, you should use Git in o
 
 The other change that lots of banks want to do is going back to the inner sourcing. Large banks, they have many IT groups all around the organization, and they want them to work together well, and the first piece is to figure out who is doing what, what resembles to what, how much code duplication you have... We have a thing that analyzes code duplication not character by character, but rather extracting the abstract syntax tree, modifying a couple things, so it's actually a very smart way of figuring out whether two pieces of code are very similar. They're so similar that if you saw them next to each other, you would say "You need to refactor them and just write one function." We're able to detect this automatically, and this helps a lot, because if you imagine you're like the CTO of a bank, and you have this codebase that dates from the '60s, and they tell you "Please put it on the cloud." That's hard. That is a harsh thing to ask from anyone.
 
-The idea of being able to tell them, "Well, actually all of this source code - let's see which parts are gonna be the easiest ones, \[unintelligible 00:21:58.18\] modernizing traditional applications", which is not really cloud-native, but we can make it be cloud-native, we can make it run in Kubernetes... And then what is the COBOL that -- you know, that's gonna be an interesting challenge to migrate. So having a view of all of this by just running a couple queries is really powerful. The other option is literally running -- I'm gonna be very hopeful and say that you could run a really huge Bash\[unintelligible 00:22:25.25\] calling Git very often, and maybe you will get something similar... But it would take hours instead of seconds.
+The idea of being able to tell them, "Well, actually all of this source code - let's see which parts are gonna be the easiest ones, there's NTA - modernizing traditional applications", which is not really cloud-native, but we can make it be cloud-native, we can make it run in Kubernetes... And then what is the COBOL that -- you know, that's gonna be an interesting challenge to migrate. So having a view of all of this by just running a couple queries is really powerful. The other option is literally running -- I'm gonna be very hopeful and say that you could run a really huge Bash\[unintelligible 00:22:25.25\] calling Git very often, and maybe you will get something similar... But it would take hours instead of seconds.
```

Comment:
Seems like this is more likely "MTA", referring to the next thing Francesc says?